src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    private List<Machine> bulldozers = new ArrayList<>();
```

Comment:
lets make this List as local variable, I don't think we need it as class-level variable. And lets create the list of Buldozers directly in get() method(same for other classes)